tests/PHPStan/Rules/Classes/ImpossibleInstanceOfRuleTest.php

```diff
@@ -167,11 +167,6 @@ public function testInstanceof(): void
 					388,
 					$tipText,
 				],
-				[
-					'Instanceof between T of Exception and Error will always evaluate to false.',
```

Comment:
Code is 
```
	/**
	 * @template T of \Exception
	 * @param T $e
	 */
	public function test(\Throwable $t, $e): void {
		if ($t instanceof $e) return;
		if ($e instanceof $t) return;
	}
```

if I call it with `test(new \Exception(), new \LogicException())` the second if is true.